src/mame/taito/taito_f3.h

```diff
@@ -337,41 +337,43 @@ class taito_f3_state : public driver_device
 	inline void alpha_blend_3b_0(u32 s);
 	inline void alpha_blend_3b_1(u32 s);
 	inline void alpha_blend_3b_2(u32 s);
-	int dpix_1_noalpha(u32 s_pix);
-	int dpix_ret1(u32 s_pix);
-	int dpix_ret0(u32 s_pix);
-	int dpix_1_1(u32 s_pix);
-	int dpix_1_2(u32 s_pix);
-	int dpix_1_4(u32 s_pix);
-	int dpix_1_5(u32 s_pix);
-	int dpix_1_6(u32 s_pix);
-	int dpix_1_8(u32 s_pix);
-	int dpix_1_9(u32 s_pix);
-	int dpix_1_a(u32 s_pix);
-	int dpix_2a_0(u32 s_pix);
-	int dpix_2a_4(u32 s_pix);
-	int dpix_2a_8(u32 s_pix);
-	int dpix_3a_0(u32 s_pix);
-	int dpix_3a_1(u32 s_pix);
-	int dpix_3a_2(u32 s_pix);
-	int dpix_2b_0(u32 s_pix);
-	int dpix_2b_4(u32 s_pix);
-	int dpix_2b_8(u32 s_pix);
-	int dpix_3b_0(u32 s_pix);
-	int dpix_3b_1(u32 s_pix);
-	int dpix_3b_2(u32 s_pix);
-	int dpix_2_0(u32 s_pix);
-	int dpix_2_4(u32 s_pix);
-	int dpix_2_8(u32 s_pix);
-	int dpix_3_0(u32 s_pix);
-	int dpix_3_1(u32 s_pix);
-	int dpix_3_2(u32 s_pix);
+	bool dpix_1_noalpha(u32 s_pix);
+	inline bool dpix_ret1(u32 s_pix);
+	inline bool dpix_ret0(u32 s_pix);
```

Comment:
these seem to be specified `inline` inconsistently despite being used in the same place.
does `inline` even mean anything here when these are used to build a function table?

Reply:
It causes the compiler not to export the function symbols.  Depending on the object format and ABI, this may or may not have any effect.